tests/e2e/tests/logging_test.py

```diff
@@ -0,0 +1,30 @@
+"""E2E test for logging API.
+
+Currently DOES NOT work.
+"""
+import json
+import logging
+import os
+import pytest
+
+from google.appengine.api.logservice import logservice
+
+
+@pytest.fixture
+def request_id():
```

Comment:
this fixture probably isn't necessary unless you plan to add a lot more tests that do this.